packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -75,19 +76,30 @@ List<CoreElement> getServiceExtensionButtons() {
   ];
 }
 
-CoreElement createTipsButton(String pageName) {
-  return PButton.icon(
-    '$pageName Docs',
-    const MaterialIcon(
-      'description',
-      Colors.black45,
-    ),
-    title: 'Documentation on using the $pageName page',
-    classes: ['btn-outline'],
-  )
-    ..small()
-    ..click(() => html.window.open(
-        'https://flutter.github.io/devtools/${pageName.toLowerCase()}', null));
+StatusItem createDocumentationStatusItem(String pageName) {
+  final icon = createIconElement(const MaterialIcon(
+    'description',
```

Comment:
use `open_in_new`?

Reply:
<img width="190" alt="screen shot 2019-02-23 at 11 13 22 am" src="https://user-images.githubusercontent.com/1226812/53290654-1e840c00-375c-11e9-874f-cecf395b0fe0.png">
I could go either way. The big advantage of open_in_new is it gives a clear signal that you can click on the link without losing your place in the application which might matter to new users.